src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -432,31 +415,11 @@ enum AppStrings {
 		static let resultCardShowResultButton = NSLocalizedString("Home_resultCard_ShowResultButton", comment: "")
 		static let resultCardNegativeTitle = NSLocalizedString("Home_resultCard_NegativeTitle", comment: "")
 		static let resultCardNegativeDesc = NSLocalizedString("Home_resultCard_NegativeDesc", comment: "")
-		static let resultCardPositiveTitle = NSLocalizedString("Home_resultCard_PositiveTitle", comment: "")
-		static let resultCardPositiveDesc = NSLocalizedString("Home_resultCard_PositiveDesc", comment: "")
-		static let resultCardPendingTitle = NSLocalizedString("Home_resultCard_PendingTitle", comment: "")
 		static let resultCardPendingDesc = NSLocalizedString("Home_resultCard_PendingDesc", comment: "")
 		static let resultCardInvalidTitle = NSLocalizedString("Home_resultCard_InvalidTitle", comment: "")
 		static let resultCardInvalidDesc = NSLocalizedString("Home_resultCard_InvalidDesc", comment: "")
 	}
-
-	enum RiskView {
-		static let unknownRisk = NSLocalizedString("unknown_risk", comment: "")
```

Comment:
these RiskView strings still are in the Localizable files

Reply:
done